server/lib/pbench/test/unit/test_process_tb.py

```diff
@@ -82,15 +83,13 @@ def mock_get(self, section: str, option: str):
                 else "Authorization-token"
             )
 
-        def mock_glob(path, pattern):
-            tbmd5 = path / "bad_log.tar.xz.md5"
-            if path == receive_path and pattern == "**/*.tar.xz.md5":
-                assert (
-                    tbmd5.suffix == ".md5"
-                ), f"Unexpected tar ball, {path!r} in mocked Path.glob()"
-                return [tbmd5]
+        def mock_glob(self: Path, pattern: str) -> List[Path]:
+            assert (
+                self == receive_path and pattern == "**/*.tar.xz.md5"
+            ), f"Unexpected tar ball, {self!r} in mocked Path.glob()"
```

Comment:
If this assertion fails, it won't be because of an "unexpected tar ball", so you should presumably change the message.  However, the assertion condition itself is complicated and expressive enough that I would just leave off the message, rather than trying to fix it (at least until we discover that we need it).

Ditto lines 132, 171, 217, and 253.